pkg/gui/custom_commands.go

```diff
@@ -151,10 +156,85 @@ func (gui *Gui) handleCustomCommandKeybinding(customCommand config.CustomCommand
 						return gui.surfaceError(err)
 					}
 
+					return gui.createMenu(title, menuItems, createMenuOptions{showCancel: true})
+				}
+			case "menuFromCommand":
```

Comment:
Now that `handleCustomCommandKeybinding` is quite large, I reckon we should split out each case into its own function. I.e. the stuff within the `menuFromCommand` should just come from a separate function. Likewise with `menu` and `input`. What do you think?

Reply:
Well, I did some novice-level refactoring :)